src/sdk/comment.ts

```diff
@@ -0,0 +1,48 @@
+import { Context } from "./context";
+import { LogReturn } from "@ubiquity-os/ubiquity-os-logger";
+import { sanitizeMetadata } from "./util";
+
+const HEADER_NAME = "Ubiquity";
```

Comment:
Confused on the purpose of this but if the full header is how we'll search for and parse the structured metadata then we can pull the org via the payload